### PR TITLE
Typechain Revived Milestone 2

### PR DESCRIPTION
**Pull request by WookashWackomy**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] This pull request is being made by the same account as the accepted application.
- [x] I have disclosed any and all sources of reused code in the submitted repositories and have done my due diligence to meet its license requirements.
- [x] In case of acceptance, invoices must be submitted and payments will be transferred to the Polkadot AssetHub and/or fiat account provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/2289 < please fill this in with the PR number of your application.

### CODE REVIEW

File: deliveries/typechain_revived_milestone_2.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+| **3**  | Handle bugs & issues                    | [Fixed in](#https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)       | Addressed all the issues and bugs we encountered while developing typechain & contributing to Abax Protocol/Governance. Additionally, we fixed all feature related issues from Brushfam's issue tracker https://github.com/Brushfam/typechain-polkadot/issues. |
+| **4**  | Incorporate Docker build into Typechain | [Implemented in](#https://github.com/C-Forge/typechain-polkadot/commit/f15c78f224d2dbdaa3af770a4f13af1316bd5e59) |                                                                                                                                                                                                                                                                |
```
Comment (semuelle) [inline deliveries/typechain_revived_milestone_2.md:25]:
```suggestion
| 0c.    | Testing and Testing Guide               | [Tests & Testing Guide](https://github.com/C-Forge/typechain-polkadot/tree/main/tests)                          |                                                                                                                                                                                                                                                                |
| 0d.    | Article                                 | [Article](https://github.com/C-Forge/typechain-polkadot/blob/main/docs/whats_new.md)                            | Covers improvements, reasonings behind them, as well as overall DX benefits.                                                                                                                                                                                   |
| **1**  | SC Deployment code                      | [Changeset](https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)      | We will have simplified & enhanced usage of deployment classes. Code's API will be consulted with the community to ensure it fits its needs.                                                                                                                   |
| **2**  | Improve Result/Error handling           | [Changeset](https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)      | Fixed several bugs & implemented correct error decoding                                                                                                                                                                                                        |
| **3**  | Handle bugs & issues                    | [Fixed in](https://github.com/C-Forge/typechain-polkadot/commit/221d891bf0f2ab63918a469bc8aaacf40579da79)       | Addressed all the issues and bugs we encountered while developing typechain & contributing to Abax Protocol/Governance. Additionally, we fixed all feature related issues from Brushfam's issue tracker https://github.com/Brushfam/typechain-polkadot/issues. |
| **4**  | Incorporate Docker build into Typechain | [Implemented in](https://github.com/C-Forge/typechain-polkadot/commit/f15c78f224d2dbdaa3af770a4f13af1316bd5e59) |                                                                                                                                                                                                                                                                |
```

Fixing hyperlinks

Comment (WookashWackomy) [reply deliveries/typechain_revived_milestone_2.md]: Thanks @semuelle and sorry about those typos.

File: deliveries/typechain_revived_milestone_2.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+| 0a.    | License                                 | MIT                                                                                                              | -                                                                                                                                                                                                                                                              |
+| 0b.    | Documentation                           | [Documentation](https://github.com/C-Forge/typechain-polkadot/blob/main/docs/about)                              | Additionally each packages contains its own extensive README.md.                                                                                                                                                                                               |
```
Comment (semuelle) [inline deliveries/typechain_revived_milestone_2.md:19]:
```suggestion
| 0b.    | Documentation                           | [Documentation](https://github.com/C-Forge/typechain-polkadot/blob/main/docs/about.md)                              | Additionally each packages contains its own extensive README.md.                                                                                                                                                                                               |
```

**Review by semuelle — APPROVED** [review]
Hi @WookashWackomy. Thanks for the delivery. I am happy to report that your milestone is hereby accepted. Congratulations, also on completing the grant. Thanks for furthering Typechain and I hope the project remains active and fruitful and we'll see you around!

**Pull request merged**